TombEngine/Objects/game_object_ids.h

```diff
@@ -175,7 +175,7 @@ enum GAME_OBJECT_ID : short
 	ID_SPHERE_OF_DOOM,
 	ID_SPHERE_OF_DOOM2,
 	ID_SPHERE_OF_DOOM3,
-	ID_CIVVIE,
+	ID_CIVVIE, //Unused
```

Comment:
This should be removed later or renamed to a missing slot. please make a issue for this one.

Reply:
Yes, once the civvy get sorted, I plan make another commit to erase thes unused civvie from TEN and from TE. (WadTool is still copying the entity on civvie).